crypto/fipsmodule/rand/rand.c

```diff
@@ -276,12 +231,6 @@ static void rand_get_seed(struct rand_thread_state *state,
                  CRNGT_BLOCK_SIZE);
 
   OPENSSL_memcpy(seed, entropy, CTR_DRBG_ENTROPY_LEN);
-
-  for (size_t i = 1; i < BORINGSSL_FIPS_OVERREAD; i++) {
-    for (size_t j = 0; j < CTR_DRBG_ENTROPY_LEN; j++) {
-      seed[j] ^= entropy[CTR_DRBG_ENTROPY_LEN * i + j];
-    }
-  }
 }
 
 #else
```

Comment:
NitPick: Please add comment indicating that this `#else` is for switching to non-fips logic. (Also for trailing `#endif`.)